test/integration-new/env_vars/env_vars_test.go

```diff
@@ -0,0 +1,180 @@
+package env_vars
+
+import (
+	"regexp"
+
+	"github.com/aws/amazon-vpc-cni-k8s/test/framework/resources/k8s/manifest"
+	k8sUtils "github.com/aws/amazon-vpc-cni-k8s/test/framework/resources/k8s/utils"
+	v1 "k8s.io/api/core/v1"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+const (
+	AWS_VPC_ENI_MTU            = "AWS_VPC_ENI_MTU"
+	AWS_VPC_K8S_CNI_LOG_FILE   = "AWS_VPC_K8S_CNI_LOG_FILE"
+	AWS_VPC_K8S_CNI_VETHPREFIX = "AWS_VPC_K8S_CNI_VETHPREFIX"
+	POD_VOL_LABEL_KEY          = "MountVolume"
+	POD_VOL_LABEL_VAL          = "true"
+)
+
+var _ = Describe("cni env test", func() {
+
+	Context("CNI Environment Variables", func() {
+		It("Verifying that secondary ENI is created", func() {
+			nodes, err := f.K8sResourceManagers.NodeManager().GetAllNodes()
+			Expect(err).NotTo(HaveOccurred())
+
+			for _, node := range nodes.Items {
+				instanceId := k8sUtils.GetInstanceIDFromNode(node)
+				instance, err := f.CloudServices.EC2().DescribeInstance(instanceId)
+				Expect(err).NotTo(HaveOccurred())
+
+				len := len(instance.NetworkInterfaces)
+				Expect(len).To(BeNumerically(">=", 2))
```

Comment:
Would this fail if the user has set WARM_ENI_TARGET=0 instead of the default which is 1, we can change this now or in next PR too.

Reply:
Yes, it depends if there are any pods behind the primary ENI. If there are no pods then number of ENIs might be just 1.